scss/modules/_results-info.scss

```diff
@@ -29,6 +29,11 @@
   margin-top: 1rem;
 }
 
+.results-info__secondary {
```

Comment:
We've been using `primary` and `secondary` to mean the blue and crimson backgrounds, respectively. For clarity, what about changing this to something like `.results-info__details` or something?